Sync WPT with upstream (21-09-2025)

servo-wpt-sync opened the pull request:
Automated downstream sync of changes from upstream as of 21-09-2025
[no-wpt-sync]

servo-wpt-sync committed:
Update web-platform-tests to revision b'1dbf4836b292d55e82bcdd23cfea6b3192e461ba'

Signed-off-by: WPT Sync Bot <ghbot+wpt-sync@servo.org>

yezhizhen commented:
Both failures have `/fetch/fetch-later/quota/same-origin-iframe/multiple-iframes.https.window.htm`. Probably should really update expectation as error instead.

yezhizhen committed:
Update multiple-iframes.https.window.js.ini

ERROR happens for two consecutive CI merge. FAIL should be the intermittent result.

Signed-off-by: Euclid Ye <yezhizhenjiakang@gmail.com>

yezhizhen commented:
https://github.com/servo/servo/tree/wpt_update_21-09-2025/tests/wpt/meta/fetch/fetch-later/quota/same-origin-iframe

I wonder if we can mark all the test in this folder as intermittent ERROR.

yezhizhen committed:
Revert changes as name is updated in intermittent tracker issue

Signed-off-by: Euclid Ye <yezhizhenjiakang@gmail.com>
